### ISSUE TITLE
[a11y-defect-3]: `va-file-input` would benefit from a border that meets or exceeds the 3:1 non-text contrast ratio success criterion

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#108676]
### Point of contact

@1copenut

### Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

### Details

Version three of the [va-file-input](https://design.va.gov/storybook/?path=/story/uswds-va-file-input--default) has a single pixel outline in #adadad that doesn't meet the 3:1 non-text contrast ratio against a white background. Reading through the [SC 1.4.11: Non-text contrast](https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html) section on "Boundaries" it reads like this isn't a contrast violation, but I feel strongly it would be an improvement to make the drop zone border 2-3 shades darker and/or 1-2 pixels thicker.

---

![Image](https://github.com/user-attachments/assets/be9af410-44b7-4d0c-b138-d33bbebdbfde)

### References, articles, or WCAG support

1. https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html

### Type of issue

- [ ] Axe-core
- [ ] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Zoom
- [x] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Focus
- [ ] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [ ] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else
